docs/features/build_from_dockerfile.md

```diff
@@ -1,6 +1,6 @@
 # Build from Dockerfile
 
-Testcontainers-go gives you the ability to build an image and run a container
+Testcontainers-Go gives you the ability to build an image and run a container
```

Comment:
@kiview I think we should standardize the naming to one of `Testcontainers-Go` or `Testcontainers-go`. 

As we are not using `Testcontainers-Java`, I'd use the lowercase version for the language.

WDYT?

Reply:
I was wondering the same when I saw the PR. In Java, if we talk about the library in a more technical context, we tend to call it `testcontainers-java`, so all lowercase.

But the general name of the project is Testcontainers, so upper cased.

I think I am with you, when talking about Testcontainers-go in a more literate way, lower casing the language looks better.

Reply:
I can lower case it then! I am glad this discussion is being had because I have created presentations to sister teams at work on this pkg, and have often wondered which were the proper route to take. 

Also, fwiw, I'll be presenting at GopherCon (US) on my team's integration testing solution which incorporates this pkg! So if there are ways of sharing that out to this community better, then I am all ears :)

Reply:
>Also, fwiw, I'll be presenting at GopherCon (US) on my team's integration testing solution which incorporates this pkg! So if there are ways of sharing that out to this community better, then I am all ears :)

@sicoyle sure, please join our [Slack](https://slack.testcontainers.org/) so that we can have a more interactive conversation about it

It's amazing that you are sharing all that great stuff you are doing at this big conference. Congrats!